pkg/paho-mqtt/include/mqtt.h

```diff
@@ -0,0 +1,208 @@
+/*
```

Comment:
Sorry to request such a change so late in the review, but I literally just noticed this... :sweat_smile:: Can we rename this file to `paho_mqtt.h` or `paho/mqtt.h` or something like this, so it does not potentially conflict with other MQTT implementations?